cluster/manifests/skipper/deployment.yaml

```diff
@@ -316,7 +316,7 @@ spec:
 {{ end }}
 {{ if or (eq .Cluster.ConfigItems.nlb_switch "pre") (eq .Cluster.ConfigItems.nlb_switch "exec") }}
           - "-forwarded-headers=X-Forwarded-For,X-Forwarded-Proto=https,X-Forwarded-Port=443"
-          - "-forwarded-headers-exclude-cidrs={{ .cluster_internal_cidrs | join "," }}"
+          - '-forwarded-headers-exclude-cidrs={{ .cluster_internal_cidrs | join "," }}'
```

Comment:
This change is not necessary as CIDRs do not contain any quotes

Reply:
Do you prefer not to have it?

Reply:
There is no difference at the end so we can keep it.